src/main/java/fr/uge/chargepointconfiguration/chargepoint/ocpp/ocpp2/data/VariableType.java

```diff
@@ -0,0 +1,9 @@
+package fr.uge.chargepointconfiguration.chargepoint.ocpp.ocpp2.data;
+
+/**
+ * Defines the variable of the machine's component.
```

Comment:
Is variable the name of the field you want to set ?

Reply:
Yeah, basically.
Will change

Reply:
Deleted